tests/test_ornithology.py

```diff
@@ -5,50 +5,63 @@
 from duck.compat import mock
 from duck.ornithology import ANY
 from duck.ornithology import Instance
+from duck.ornithology import Needle
 
 
-def test_ornithology():
+def test_ornithology_any():
+    """
+    Tests for ANY alias.
+    """
     mockint = duck.Mock(spec=int)
-    mockfloat = duck.Mock(spec=float)
-    assert Instance(int) == mockint
-    assert Instance(int) != mockfloat
-    assert Instance(int) == 42
-    assert Instance(int) != 2.718
     assert ANY == mockint
     assert not ANY != mockint
     assert ANY == "foo"
     assert not ANY != "foobar"
-    assert repr(Instance(int)) == '<Instance: int>'
+    # Reverse order tests
```

Comment:
We should _either_ remove these comments, or wrap it in `if sys.version_info < (3, 5):`

My vote is outright removal.